config/routes.rb

```diff
@@ -42,14 +42,15 @@
           post :transfer_multiple
           post :confirm_multiple
           post :cancel_multiple
+          put :cancel_multiple
```

Comment:
any reason there are both `post` and `put` routes for the cancel_multiple action? Can we get rid of one or the other?